src/tools/ceph-dencoder/osd_types.h

```diff
@@ -3,6 +3,7 @@ TYPE(osd_info_t)
 TYPE_FEATUREFUL(osd_xinfo_t)
```

Comment:
How about decoupling OSD from MDS? Now this PR needs reviews from 2 components.

Reply:
will do that, i thought MDS changes won't be too much, but now it is reasonable to decoupling them